### PR TITLE
Stability improvements and bug fixes

### PR DESCRIPTION
**Pull request by SouthAlbertaAI**
This includes some bug fixes and the early pieces of the future alert mode and 511 alberta road conditions commands

I'll continue to take care of the alert mode and road conditions pieces as I have a specific view for them even though they are committed in a skeleton state for now.

Expect a PR for the alert and road conditions parts soon!

In addition: You will see a pre commit yaml file. This is for future implementation but I use its checks in my current environment to format code before its commit. This can be configured to run automatically using the pre-commit library

### CODE REVIEW

**Comment by glob-bruh** [comment]
Will check tonight.

**Comment by SouthAlbertaAI** [comment]
Sounds good, just updated PR to remove some old breakpoint bug testing code

File: static/CrossJoin_CommandLogic.py [hunk 1]
```diff
@@ -337,29 +342,41 @@ def GetCams(user_input: str):
...
         return Sys.ErrorMessage_Command(str(e))
 
 
-def SendHelp(user_input: str):
+def GetRoadConditions(user_input: str = None):
+    headers = {
+        "content-type": "application/json"
+    }
+    return_text = r.get(
+        "https://511.alberta.ca/api/v2/get/winterroads",
```
Comment (glob-bruh) [inline static/CrossJoin_CommandLogic.py]: `https://511.alberta.ca/api/v2/get/winterroads`
should be 
`https://511.alberta.ca/api/v2/get/winterroads?format=json&lang=en`

**Review by glob-bruh — CHANGES_REQUESTED** [review]
The 511 API call should have the optional parameters included.



> https://511.alberta.ca/api/v2/get/winterroads
> should be
> https://511.alberta.ca/api/v2/get/winterroads?format=json&lang=en

If you dont want to change this then it can be pushed through.

**Comment by SouthAlbertaAI** [comment]
There we go changes are all made

**Review by glob-bruh — APPROVED** [review]

**Pull request merged**